src/pages/navigation.js

```diff
@@ -0,0 +1,28 @@
+import { Navigation, ImgLogo, Menu, MenuList, Span, Hamburger, CloseIcon } from "../style/navigation/styled";
+import logo from "../assets/shared/logo.svg";
+import { Link } from "react-router-dom";
+import hamburger from "../assets/shared/icon-hamburger.svg";
+import { useState} from "react";
+import closeIcon from "../assets/shared/icon-close.svg";
+
+export const Nav = () => {
+  const [navOpen, setNavOpen] = useState(false);
+
+  return (
+      <>
+    <Navigation>
+            <ImgLogo src={logo} alt="Space logo" />
+        <Menu open={navOpen}>
+          <CloseIcon src={closeIcon}></CloseIcon>
+                <MenuList className="menu-list">
+                    <Link to="/space-tourism/" className="menu-item"><Span><strong>00</strong> HOME</Span></Link>
```

Comment:
You shouldn't have a component name as `Span` since span is a html tag and it has no meaning